cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -99,6 +109,7 @@ pub(crate) struct YaccParser {
     num_newlines: usize,
     ast: GrammarAST,
     global_actiontype: Option<String>,
+    duplicate_rule_spans: HashMap<Span, Vec<Span>>,
```

Comment:
Let's add a comment/docstring that explains what the two `Span`s mean (otherwise we'll forget!).

Reply:
fixed in https://github.com/softdevteam/grmtools/pull/307/commits/a39ed7e4baf5a69a04964daaed69c0667d0ee0ae